ci-operator/step-registry/upi/conf/libvirt/network/upi-conf-libvirt-network-commands.sh

```
@@ -24,32 +24,18 @@ if [[ ! -f "${CLUSTER_PROFILE_DIR}/leases" ]]; then
-}
-
-if [ "${USE_EXTERNAL_DNS:-false}" == "true" ]; then
-  BASE_DOMAIN="phc-cicd.cis.ibm.net"
-  CLUSTER_NAME="${LEASED_RESOURCE}"
-else
-  BASE_DOMAIN="${LEASED_RESOURCE}.ci"
-  CLUSTER_NAME="${LEASED_RESOURCE}-${UNIQUE_HASH}"
-fi
-BASE_URL="${CLUSTER_NAME}.${BASE_DOMAIN}"
+# shellcheck source=../../libvirt/cluster-context/upi-libvirt-cluster-context-commands.sh
+source "$(cd "$(dirname "${BASH_SOURCE[0]}")/../../libvirt/cluster-context" && pwd)/upi-libvirt-cluster-context-commands.sh"
```

Shellcheck (0.11.0)

[info] Not following: ../../libvirt/cluster-context/upi-libvirt-cluster-context-commands.sh was not specified as input (see shellcheck -x).

(SC1091)